spec/policies/event_policy_spec.rb

```diff
@@ -3,21 +3,65 @@
 RSpec.describe EventPolicy do
 
   let(:reviewer) { create(:user, :reviewer) }
+  let(:program_team) { create(:user, :program_team) }
   let(:organizer) { create(:organizer) }
   let(:admin) { create(:admin) }
   let(:event) { create(:event) }
 
   subject { described_class }
 
-  #permissions ".scope" do
-  #  pending "add some examples to (or delete) #{__FILE__}"
-  #end
+  permissions ".scope" do
```

Comment:
Haha, I thought scopes were saying it was for groups of items like @events = Event.all! Confused by the docs and thinking it was some weird way to configure access to index views. Taken out now.